sites/NOAA_GFDL/mdtf_frepp_template.jsonc

```diff
@@ -0,0 +1,137 @@
+// Configuration for MDTF-diagnostics driver script.
+//
+// All text to the right of an unquoted "//" is a comment and ignored, as well
+// as blank lines (JSONC quasi-standard.)
+//
+// Copy this file and customize the setting values as needed. Pass it to the
+// framework at the end of the command line (positionally) or with the
+// -f/--config-file flag.
+//
+{
+  // The cases below correspond to the different test data sets.
+  //
+  // Note that the mdtf package does not yet handle multiple cases.
+  //
+  "case_list" : [
+    {
+      "CASENAME" : "CASENAME1",
+      "model" : "MODEL1",
+      "convention" : "CONVENTION1",
+      "experiment" : "CASENAME1",
+      "FIRSTYR" : FIRSTYR1,
+      "LASTYR" : LASTYR1,
+      "CASE_ROOT_DIR" : "PPDIR1/",
+      "pod_list": [
+              POD_LIST
+      ]
+    }
+  ],
+  // PATHS ---------------------------------------------------------------------
+  // Location of input and output data. If a relative path is given, it's
+  // resolved relative to the MDTF-diagnostics code directory.
+  // set this to somewhere writeable with sufficient space for copies of input data
+  //"MDTF_TMPDIR": "/net2/jml/tmp",
+  // Parent directory containing results from different models.
+  "MODEL_DATA_ROOT": "$MDTF_TMPDIR/inputdata/model",
+
+  // Site-specific installation of observational data used by individual PODs.
+  // This will be GCP'ed locally if running on PPAN.
+  //"OBS_DATA_REMOTE": "/home/oar.gfdl.mdtf/mdtf/inputdata/obs_data",
+  "OBS_DATA_REMOTE": "/archive/wnd/MDTF-GFDL/inputdata/obs_data",
+
+  // Parent directory containing observational data used by individual PODs.
+  "OBS_DATA_ROOT": "$MDTF_TMPDIR/inputdata/obs_data",
+
+  // Working directory.
+  "WORKING_DIR": "$MDTF_TMPDIR/wkdir",
+
+  // Directory to write output files. Defaults to working directory if blank.
+  //"OUTPUT_DIR": "$MDTF_TMPDIR/mdtf_out",
+  "OUTPUT_DIR": "OUTPUTDIR1",
+
+  // GFDL ----------------------------------------------------------------------
+  // Settings specific to operation at GFDL.
+
+  // If running on GFDL PPAN, set $MDTF_TMPDIR to this location and
+  // create temp files here. Must be accessible via gcp.
+  "GFDL_PPAN_TEMP": "$TMPDIR",
+
+  // If running on a GFDL workstation, set $MDTF_TMPDIR to this location
+  // and create temp files here. Must be accessible via gcp.
+  //"GFDL_WS_TEMP": "$MDTF_TMPDIR",
+  "GFDL_WS_TEMP": "/archive/$USER/tmp",
+
+  // Set flag to run framework in 'online' mode, processing data as part of the
+  // FRE pipeline. Normally this is done by calling the mdtf_gfdl.csh wrapper
+  // script from the XML.
+  "frepp": false,
+
```

Comment:
```suggestion

// Allow the framework to pull files/variables from multiple components
"any_components": true,

```

Reply:
I just came across this JSON option today.  In order for PODs to operate across components, this flag will need to be set.  Might as well set this now.